### DISCUSSION TITLE
Block links currently do not function

### DISCUSSION THREAD

**Discussion by aliceinblunderland0** [alangrainger/share-note#47] [Vote for upcoming features]
Hello. This is a wonderfully useful plugin. I did come across a hiccup, and that involves block links. I very often link to blocks across notes (so for example, adding a link to a specific section of note A in the contents of note B). However, when both notes exported and not encrypted, if from note B I open that link, I'm brought at the top of note A. A way to somehow make it clear what block I'm talking about would be extremely practical, whether it's moving the selection there, or highlighting it, or similar. Thank you for your time and your plugin!

**Comment by alangrainger** [comment]
Regarding block linking, I'll leave this ticket open, but it doesn't look promising:

If you switch to Reading View and look at the document source, you will see that the block link identifiers (the `^somelink` part) disappear and there's no record of where they should be.

I don't know if there's any way to find out where their location would be in the rendered document, so right now it doesn't look hopeful.

Instead, as an option you could link to the **Heading** in note A, rather than a block. I'll make sure inter-note linking to headings is functional.
